### PR TITLE
Update link_content_credential_phishing.yml

### PR DESCRIPTION
**Pull request by morriscode**
Various legitimate docusign negations

### CODE REVIEW

File: detection-rules/link_content_credential_phishing.yml [hunk 1]
```diff
@@ -39,7 +39,20 @@ source: |
...
   )
-
+  
+  // negate docusign 'via' messages
+  and not (
+      any(headers.hops,
+          any(.fields, .name == "X-Api-Host" and strings.ends_with(.value, "docusign.net"))
+      )
+      and strings.contains(sender.display_name, "via")
+    )
```
Comment (jkamdjou) [inline detection-rules/link_content_credential_phishing.yml:49]: i think indentation is wrong here

**Review by jkamdjou — APPROVED** [review]
🙌 do we want this on the other docusign rules too?

File: detection-rules/link_content_credential_phishing.yml (hunk 1 above)
Comment (morriscode) [reply detection-rules/link_content_credential_phishing.yml:49]: Does it look better now? This is how the formatter formatted it :)

**Comment by morriscode** [comment]
> 🙌 do we want this on the other docusign rules too?

The via component is in a couple other docusign rules already. I'll see if it's needed to add the received server raw component.

**Pull request merged**